module/x/gravity/keeper/invariants.go

```diff
@@ -58,10 +63,16 @@ func ModuleBalanceInvariant(k Keeper) sdk.Invariant {
 
 			// Collect the send amount + fee amount for each tx
 			txTotal := tx.Erc20Token.Amount.Add(tx.Erc20Fee.Amount)
+			_, ok := expectedBals[denom]
```

Comment:
A comment here about what exactly how this logic works would be good. Ah I see you cleaned it up the next commit, great! NVM